torch/ao/sparsity/experimental/activation_sparsifier/activation_sparsifier.py

```diff
@@ -0,0 +1,149 @@
+from typing import Dict, Any
+from collections import defaultdict
+from torch import nn
+
+__all__ = ['ActivationSparsifier']
+
+
+class ActivationSparsifier:
+    r"""
+    The Activation sparsifier class aims to sparsify/prune activations in a neural
+    network. The idea is to attach the sparsifier to a layer (or layers) and it
+    zeroes out the activations based on the mask_fn (or sparsification function)
+    input by the user.
+    The mask_fn is applied once all the inputs are aggregated and reduced i.e.
+    mask = mask_fn(reduce_fn(aggregate_fn(activations)))
+
+    Note::
+        The sparsification mask is computed on the input **before it goes through the attached layer**.
+
+    Args:
```

Comment:
Please, follow the sphinx syntax: https://www.sphinx-doc.org/en/master/usage/extensions/napoleon.html

- No blanks between args
- Args should end with a colon

Reply:
done